### PR TITLE
load from csv component

### PR DESCRIPTION
**Pull request by Hakimovich99**

### CODE REVIEW

File: src/components/load_from_csv/fondant_component.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Load from csv file
+description: Component that loads a dataset from huggingface hub
+image: hakimovich99/load_from_csv:dev #TODO: change it to fndnt once implemented
```
Comment (RobbeSneyders) [inline src/components/load_from_csv/fondant_component.yaml:3]: You can use the `ml6team` github container registry for any components related to a use case.

File: src/components/load_from_csv/fondant_component.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Load from csv file
+description: Component that loads a dataset from huggingface hub
```
Comment (RobbeSneyders) [inline src/components/load_from_csv/fondant_component.yaml:2]: Should be updated.

File: src/components/text_cleaning/__init__.py
Comment (RobbeSneyders) [inline src/components/text_cleaning/__init__.py:1]: This file should not be in here.

**Review by RobbeSneyders — COMMENTED** [review]
Thanks @Hakimovich99! Looks good apart from some minor comments.

File: src/components/text_cleaning/__init__.py
Comment (Hakimovich99) [reply src/components/text_cleaning/__init__.py:1]: The pre-commit check doesn't work if I don't set this file: `src/components/text_cleaning/src/main.py: error: Duplicate module named "main" (also at "src/components/load_from_csv/src/main.py")`

File: src/components/load_from_csv/fondant_component.yaml (hunk 1 above)
Comment (Hakimovich99) [reply src/components/load_from_csv/fondant_component.yaml:3]: Just to make sure, like that: `ghcr.io/ml6team/load_from_csv:dev` ?

Comment (RobbeSneyders) [reply src/components/load_from_csv/fondant_component.yaml:3]: Yes. You can label them to link them to this repo. 
See [this script](https://github.com/ml6team/fondant-usecase-datacomp/blob/main/scripts/build_components.sh) for instance, it uses the `fondant build` command.

Comment (Hakimovich99) [reply src/components/load_from_csv/fondant_component.yaml:3]: Thanks!

File: src/components/text_cleaning/__init__.py
Comment (RobbeSneyders) [reply src/components/text_cleaning/__init__.py:1]: Yes, the mypy pre-commit doesn't work well in a multi-repo. However adding the `__init__.py` files is only a band-aid and will not solve it. Feel free to remove the mypy pre-commit, we did it in other use case repositories as well. Eg. [here](https://github.com/ml6team/fondant-usecase-datacomp/commit/e6985f9ba43e1080b5bde024c1803a9ca1805377).

File: src/weaviate_hakim/docker-compose.yml
Comment (RobbeSneyders) [inline src/weaviate_hakim/docker-compose.yml:1]: I don't think this file needs to be in here?

Comment (Hakimovich99) [reply src/weaviate_hakim/docker-compose.yml:1]: Indeed thanks!

**Review by RobbeSneyders — APPROVED** [review]
Thanks @Hakimovich99!

**Pull request merged**